content/posts/android-tv-11.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "New Android TV Release - v0.11.0"
+subtitle = "Improved Video Playback That Goes up to 11"
+author = "Niels van Velzen"
+githubusername = "NielsvanVelzen"
+date = "2020-02-04"
+justify = "center"
++++
+
+Thank you for using Jellyfin! This is a major update for the Android TV client.
+
+<!--more-->
+
```

Comment:
Might need to remove the newlines here to avoid odd spacing in the preview.

Reply:
Doesn't make a difference from my testing.